app/templates/briefs/brief_response.html

```diff
@@ -0,0 +1,37 @@
+{% extends "services/_base_edit_section_page.html" %}
+
+{% block breadcrumb %}
+
+  {%
+    with items = [
+      {
+        "link": "/",
+        "label": "Digital Marketplace"
+      },
+      {
+        "link": "/{}/opportunities".format(brief.frameworkSlug),
+        "label": "Supplier opportunities"
+      },
+      {
+        "link": "/{}/opportunities/{}".format(brief.frameworkSlug, brief.id),
+        "label": brief.title
+      },
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block save_button %}
+
+  {%
+    with
+    label="Save and continue",
```

Comment:
@ralph-hawkins wanted to rename this button to "Apply for this opportunity".
Since you've split this template, this should now be easy to do.